src/main/java/org/jenkinsci/plugins/workflow/multibranch/SCMBinder.java

```diff
@@ -165,22 +167,4 @@ public SCMBinder(String scriptPath) {
 
     }
 
-    // TODO seems there is no general-purpose ConsoleNote which simply wraps markup in specified HTML
-    @SuppressWarnings("rawtypes")
-    public static class WarningNote extends ConsoleNote {
```

Comment:
`@Restricted(NoExternalUse.class)`?

Reply:
I do not see why; it was not used elsewhere to begin with.

Reply:
That said, removing this class might cause trouble with old build logs containing non-deserializable annotations? Or is this going somewhere else?

Reply:
https://github.com/jenkinsci/jenkins/blob/aa83562568b44aa85ff17bdfab831bbe39c2fd15/core/src/main/java/hudson/console/ConsoleAnnotationOutputStream.java#L112-L125 should just drop the annotation, i.e. make the text be plain rather than yellow. Not specifically tested.